### PR TITLE
Adding app to private store

### PR DESCRIPTION
**Pull request by joinimran**
I have updated the content to make apps available in a private store.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2109

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: store-for-business/distribute-apps-from-your-private-store.md [hunk 1]
```diff
@@ -43,13 +43,11 @@ Microsoft Store adds the app to **Products and services**. Click **Manage**, **A
 
     <!--- ![Image showing Manage menu in Microsoft Store for Business.](images/wsfb-manageinventory.png) -->
 
-3.  Use **Refine results** to search for online-licensed apps under **License type**.
-4.  From the list of online-licensed apps, click the ellipses for the app you want, and then choose **Add to private store**.
+3.  Click on the application to open the application settings and then click on **Private store availability**.
```
Comment (nenonix) [inline store-for-business/distribute-apps-from-your-private-store.md]:
```suggestion
3.  Click on the application to open the application settings, then select **Private store availability**.
```

**Comment by joinimran** [comment]
Copy review suggestions applied.

**Comment by mypil** [comment]
@TrudyHa - PR has been copy edited and is ready for final review. This is based on issue #2109 

Thank you.

**Pull request merged**